src/lib/plugins/run-code-blocks/server.ts

```diff
@@ -1,5 +1,6 @@
 import { ChildProcess, spawn } from 'child_process';
 import { Option, assert } from 'ts-std';
+import { stripVTControlCharacters } from 'node:util';
```

Comment:
noice

Reply:
I went to use `strip-ansi` and found that they have "soft deprecated" the package because node built it in 🎉